### PR TITLE
Inactive

### PR DESCRIPTION
**Pull request by seanmtracey**
Added inactive state for 'size' input when theme is none

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ftlabs-email-signatures",
-  "version": "0.0.1",
+  "version": "1.2.0",
```
Comment (JakeChampion) [inline package.json]: That is a big bump. Why are we jumping passed 1.0.0?

File: src/dialog.html [hunk 2]
```diff
@@ -44,6 +44,11 @@
 			transform: rotateX(-90deg);
 		}
 
+		*[data-clickable="false"]{
+		    opacity: 0.3;
+		    cursor: not-allowed !important;
```
Comment (JakeChampion) [inline src/dialog.html]: `!important`?

File: src/scripts/popup.js [hunk 3]
```diff
@@ -41,6 +41,24 @@ justOnce.addEventListener('click', function (e) {
 	});
 });
 
+size.addEventListener('mousedown', function(e){
+	console.log(this.getAttribute('data-clickable'));
```
Comment (JakeChampion) [inline src/scripts/popup.js]: Remove this?

File: src/scripts/popup.js [hunk 4]
```diff
@@ -41,6 +41,24 @@ justOnce.addEventListener('click', function (e) {
...
+	if(this.getAttribute('data-clickable') === 'false'){
+		e.preventDefault();
+	}
+
+}, false);
+
+theme.addEventListener('change', function(){
+	// console.log(this.value);
```
Comment (JakeChampion) [inline src/scripts/popup.js]: Remove this?

File: src/scripts/popup.js [hunk 5]
```diff
@@ -41,6 +41,24 @@ justOnce.addEventListener('click', function (e) {
 	});
 });
 
+size.addEventListener('mousedown', function(e){
+	console.log(this.getAttribute('data-clickable'));
+	if(this.getAttribute('data-clickable') === 'false'){
+		e.preventDefault();
```
Comment (JakeChampion) [inline src/scripts/popup.js:47]: Do we want to stop propagation as well?

File: package.json (hunk 1 above)
Comment (seanmtracey) [inline package.json]: We went past 1.0.0 a little while ago (live version is at 1.1.0) for some reason this hasn't made it's way into the git repos

File: src/dialog.html (hunk 2 above)
Comment (seanmtracey) [inline src/dialog.html]: Yeah, I wasn't happy with this either. It was a placeholder that made it further than it should have. Will look into why the style won't apply without it.

Comment (seanmtracey) [inline src/dialog.html]: The Origami CSS overrides this property. Moving the < link > to before the < style > removes the need for !important.

File: src/scripts/popup.js (hunk 5 above)
Comment (seanmtracey) [inline src/scripts/popup.js:47]: Behaves itself without it, but sure, I see no reason not too.

File: package.json (hunk 1 above)
Comment (JakeChampion) [inline package.json]: How are you bumping this? `npm version`?

File: src/dialog.html [hunk 6]
```diff
@@ -1,6 +1,9 @@
 <html>
 <head>
-		
+	
+	<!-- <link rel="stylesheet" href="//build.origami.ft.com/bundles/css?modules=o-forms@^2.0.0,o-buttons@^3.0.3" /> -->
```
Comment (JakeChampion) [inline src/dialog.html:3]: Remove this?

File: src/manifest.json [hunk 7]
```diff
@@ -3,7 +3,7 @@
 
   "name": "FTLabs GMail Signatures",
   "description": "A Chrome Extension that automagically adds links from RSS feeds to the end of emails sent in the Gmail web client",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (JakeChampion) [inline src/manifest.json:6]: This seems like the correct version number. I'd remove the one in `package.json`

Comment (seanmtracey) [inline src/manifest.json:6]: Ah, that's right. Didn't notice it was package.json and not manifest.json.

File: src/dialog.html (hunk 6 above)
Comment (seanmtracey) [inline src/dialog.html:3]: I think it's useful to have the URL we used to generate the shrinkwrapped URL available.

**Comment by JakeChampion** [comment]
LGTM

**Pull request merged**